packages/widget_toolkit_pin/pubspec.yaml

```diff
@@ -31,9 +32,9 @@ dev_dependencies:
     sdk: flutter
   golden_toolkit: ^0.14.0
   mockito: ^5.3.2
-  rx_bloc_generator: ^6.0.1
+  rx_bloc_generator: ^6.0.2
```

Comment:
Is there a reason we do not stick to the established strategy for adding dependencies?

Reply:
I have returned the version to rx_bloc_generator: ^6.0.1

Reply:
Do you need `6.0.1` ? If not you can use 6.0.0 as discussed

Reply:
I have set it to 6.0.0